classes/item.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../modules/date_handler'
+class Item
+  attr_accessor :publish_date, :genre, :author, :source, :label
+  attr_reader :id, :archived
+
+  def initialize(publish_date:, id: nil, archived: false)
+    @id = id || Random.rand(1...1000)
+    @publish_date = DateHandler.from_string(publish_date)
+    @archived = archived
+  end
```

Comment:
- [ ] you also need to add the one-to-many relationship in the Item Class, please have another look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more details